src/source/PeerConnection/Rtp.c

```diff
@@ -71,12 +49,59 @@ STATUS createKvsRtpTransceiver(RTC_RTP_TRANSCEIVER_DIRECTION direction, DOUBLE r
     return retStatus;
 }
 
+STATUS setUpRollingBufferConfigInternal(PKvsRtpTransceiver pKvsRtpTransceiver, PRtcMediaStreamTrack pRtcMediaStreamTrack,
+                                        DOUBLE rollingBufferDurationSec, DOUBLE rollingBufferBitratebps)
+{
+    STATUS retStatus = STATUS_SUCCESS;
+    CHK_ERR(pKvsRtpTransceiver != NULL || pRtcMediaStreamTrack != NULL, STATUS_NULL_ARG,
+            "Media track and transceiver not set. Make sure to set up transceiver with addTransceiver()");
+    if (pKvsRtpTransceiver->pRollingBufferConfig == NULL) {
+        pKvsRtpTransceiver->pRollingBufferConfig = (PRollingBufferConfig) MEMCALLOC(1, SIZEOF(RollingBufferConfig));
+        CHK(pKvsRtpTransceiver->pRollingBufferConfig != NULL, STATUS_NOT_ENOUGH_MEMORY);
+    }
+
+    if (rollingBufferDurationSec >= MIN_ROLLING_BUFFER_DURATION_IN_SECONDS && rollingBufferDurationSec <= MAX_ROLLING_BUFFER_DURATION_IN_SECONDS) {
+        pKvsRtpTransceiver->pRollingBufferConfig->rollingBufferDurationSec = rollingBufferDurationSec;
+    } else {
+        DLOGW("Rolling buffer duration does not fit range (%lf sec - %lf sec). Setting to default %lf sec", MIN_ROLLING_BUFFER_DURATION_IN_SECONDS,
+              MAX_ROLLING_BUFFER_DURATION_IN_SECONDS, DEFAULT_ROLLING_BUFFER_DURATION_IN_SECONDS);
```

Comment:
You should add what the current value is set to. Also, should add "audio" and "video" for the duration lines.

The below logs are missing that info..
```
2024-06-28 23:40:41.721 WARN    setUpRollingBufferConfigInternal(): Rolling buffer duration does not fit range (0.100000 sec - 10.000000 sec). Setting to default 3.000000 sec
2024-06-28 23:40:41.721 WARN    setUpRollingBufferConfigInternal(): Rolling buffer bitrate does not fit range (104857.600000 bps - 251658240.000000 bps) for audio. Setting to default 1024000.000000 bps
2024-06-28 23:40:41.722 WARN    setUpRollingBufferConfigInternal(): Rolling buffer duration does not fit range (0.100000 sec - 10.000000 sec). Setting to default 3.000000 sec
2024-06-28 23:40:41.722 WARN    setUpRollingBufferConfigInternal(): Rolling buffer bitrate does not fit range (104857.600000 bps - 251658240.000000 bps) for video. Setting to default 5242880.000000 bps
```